mail/2.0/_index.md

```diff
@@ -1,24 +1,24 @@
 ---
-title: "Jakarta Mail 2.0 (milestone)"
-date: 2020-06-05
+title: "Jakarta Mail 2.0"
+date: 2020-08-25
 summary: "Release for Jakarta EE 9"
 ---
 
 Jakarta Mail defines a platform-independent and protocol-independent framework to build mail and messaging applications.
 
 * [Jakarta Mail 2.0 Release Record](https://projects.eclipse.org/projects/ee4j.mail/releases/2.0.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Mail 2.0 Specification Document](./mail-spec-2.0-RC6.pdf) (PDF)
-* [Jakarta Mail 2.0 Specification Document](./mail-spec-2.0-RC6.html) (HTML)
+* [Jakarta Mail 2.0 Specification Document](./mail-spec-2.0.pdf) (PDF)
```

Comment:
shouldn't it have jakarta- in the file name?
Same for html bellow

Reply:
PDF of form {spec}-spec-x.y.pdf ("-spec" preferred but not required) is the requirement, where spec is "mail" in this case. `jakarta-` prefix is not mandatory thing for this release, even though I've added it.